### DISCUSSION TITLE
[Bug report] Editorial Theme having Faint Font Color

### DISCUSSION THREAD

**Discussion by VincentNyoike** [GetPublii/Publii#1430] [Bug Report]
### Operating system

Windows

### Publii version

0.45.2 (build 16609)

### Issue type

Free theme

### Bug description

It is not a bug as such, but the font color in Editorial theme is not clear. I have noted that people with partial blindness cannot read a text on Editorial theme because it is too faint. 

### Steps to reproduce

Please make Editorial theme font color clear (darker).

**Comment by internettips** [comment]
You're right, this isn't a bug. Perhaps, also much depends on which theme you're using: they have quite a few 'start-off' theme varieties at: https://demo.getpublii.eu/themes/editorial/ . 

Though when I used Editorial theme on a previous project -- with accessibility a top priority -- I used the 'Light' theme at: https://demo.getpublii.eu/themes/editorial/light/ as a basis, then added some custom CSS rules to essentially make all text black or near black when displayed on a white or light background. 

In addition, thinking back, I probably increased text size as well for some small text items. Just a few tweaks like this and the end result was Accessibility scores of 100/100 when measured by Google Lighthouse Accessibility tool.

Of course for folks who use a screen reader, most of the ARIA codes probably already come built-in to Publii themes -- they have mastered this aspect with remarkable attention to detail.

**Reply by candidexmedia** [reply]
These are great tips @internettips!

I just realized that there are *two* Editorial Themes, and both are called "Editorial" on their respective pages. I assume OP is talking about the free one, but want to make sure before giving CSS snippets.

@VincentNyoike which Theme are you referring to?

Editorial 2: https://marketplace.getpublii.com/themes/editorial-2/

![](https://marketplace.getpublii.com/wp-content/uploads/2020/10/editorial-2-1-768x966.jpg)

OR

Editorial: https://marketplace.getpublii.com/themes/editorial/

![](https://marketplace.getpublii.com/wp-content/uploads/2019/03/editorial-768x966.jpg)

**Reply by VincentNyoike** [reply]
I am referring to the first one.

**Comment by VincentNyoike** [comment]
> You're right, this isn't a bug. Perhaps, also much depends on which theme you're using: they have quite a few 'start-off' theme varieties at: https://demo.getpublii.eu/themes/editorial/ .
> 
> Though when I used Editorial theme on a previous project -- with accessibility a top priority -- I used the 'Light' theme at: https://demo.getpublii.eu/themes/editorial/light/ as a basis, then added some custom CSS rules to essentially make all text black or near black when displayed on a white or light background.
> 
> In addition, thinking back, I probably increased text size as well for some small text items. Just a few tweaks like this and the end result was Accessibility scores of 100/100 when measured by Google Lighthouse Accessibility tool.
> 
> Of course for folks who use a screen reader, most of the ARIA codes probably already come built-in to Publii themes -- they have mastered this aspect with remarkable attention to detail.

**Comment by VincentNyoike** [comment]
Is it something you can help me with if you don't mind? I am not an expert in CSS stuff.

**Reply by candidexmedia** [reply]
Bob, one of the Publii devs, provided an updated version of the theme. 😊

That said, if you still want to tweak the text colour, go to `Tools & Plugins > Custom CSS` in Publii and paste the following:

```css
body, input, select, textarea {
	color: #6e767c;
}
```

Change the `#6e767c` [color value](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) with the HEX, RGB, HSL, hwb, lab, lch named colour, etc. of your choice.

**Comment by bobmitro** [comment]
The Editorial 2 theme was created by @ajlkn for HTML5 UP. That's why we didn't interfere with the styling; we just adapted it to Publii's CMS. 

However, I have adjusted the font color contrast ratio to ensure a minimum contrast ratio of 4.5:1. Just download the [Editorial 2 v.1.4.1.0 ](https://marketplace.getpublii.com/themes/editorial-2/) and update your installation.
